plasmapy/utils/decorators/tests/test_validators.py

```diff
@@ -29,40 +37,22 @@ class :class:`~plasmapy.utils.decorators.validators.ValidateQuantities`.
     )  # type: Dict[str, Any]
     check_defaults = {**unit_check_defaults, **value_check_defaults}
 
-    @staticmethod
-    def foo(x):
-        return x
-
-    @staticmethod
-    def foo_anno(x: u.cm):
-        return x
-
     def test_inheritance(self):
         assert issubclass(ValidateQuantities, CheckUnits)
         assert issubclass(ValidateQuantities, CheckValues)
```

Comment:
```suggestion
    @pytest.mark.parametrize("_class", [CheckUnits, CheckValues])
    def test_inheritance(self, _class):
        assert issubclass(ValidateQuantities, _class)
```

We ought to parametrize this test method as well.